pennylane/math/__init__.py

```diff
@@ -89,11 +89,17 @@
     cast,
     cast_like,
     convert_like,
-    get_deep_interface,
-    get_interface,
     in_backprop,
-    is_abstract,
     requires_grad,
+    is_abstract,
+)
+from .interface_utils import (
+    _get_canonical_interface_name,
```

Comment:
Potentially time to get rid of the `_` on the name and make the helper public.

Reply:
True. Resolving!